apps/user-ms/src/main/java/it/pagopa/selfcare/user/controller/UserController.java

```diff
@@ -227,13 +231,17 @@ public Uni<UsersNotificationResponse> getUsers(@QueryParam(value = "page") @Defa
     @Produces(MediaType.APPLICATION_JSON)
     public Multi<UserInstitutionResponse> retrievePaginatedAndFilteredUser(@QueryParam(value = "institutionId") String institutionId,
                                                                            @QueryParam(value = "userId") String userId,
-                                                                           @QueryParam(value = "roles") List<PartyRole> roles,
+                                                                           @QueryParam(value = "roles") List<String> roles,
                                                                            @QueryParam(value = "states") List<String> states,
                                                                            @QueryParam(value = "products") List<String> products,
                                                                            @QueryParam(value = "productRoles") List<String> productRoles,
                                                                            @QueryParam(value = "page") @DefaultValue("0") Integer page,
                                                                            @QueryParam(value = "size") @DefaultValue("100") Integer size) {
-        return userService.findPaginatedUserInstitutions(institutionId, userId, roles, states, products, productRoles, page, size);
+        List<PartyRole> roleList = new ArrayList<>();
+        if(!CollectionUtils.isNullOrEmpty(roles)) {
+            roleList.addAll(roles.stream().map(PartyRole::valueOf).toList());
```

Comment:
you could check if exists any PartyRole with that value and in case throw an 400 exception